sequencer/node/src/client.rs

```diff
@@ -147,7 +160,7 @@ impl Client {
                 };
 
                 if self.size < tx.len() {
-                    warn!("Transaction size too big");
+                    warn!("Transaction size too big. Tx size is: {}", tx.len());
                     break 'main;
                 }
                 tx.resize(self.size, 0u8);
```

Comment:
I believe these lines can be removed without any consequences. Please try and if so, let's remove them.